scripts/globals/abilities/pets/hastega.lua

```diff
@@ -1,26 +1,26 @@
 ---------------------------------------------
 -- Hastega
 ---------------------------------------------
-require("scripts/globals/monstertpmoves");
-require("scripts/globals/settings");
-require("scripts/globals/status");
-require("scripts/globals/utils");
-require("scripts/globals/msg");
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/utils")
+require("scripts/globals/msg")
 ---------------------------------------------
 
 function onAbilityCheck(player, target, ability)
-    return 0,0;
-end;
+    return 0, 0
+end
 
 function onPetAbility(target, pet, skill, summoner)
-    local bonusTime = utils.clamp(summoner:getSkillLevel(dsp.skill.SUMMONING_MAGIC) - 300, 0, 200);
+    local bonusTime = utils.clamp(summoner:getSkillLevel(dsp.skill.SUMMONING_MAGIC) - 300, 0, 200)
     local duration = 180 + bonusTime;
     print(bonusTime)
     print(duration)
 
     -- Garuda's Hastega is a weird exception and uses 153 instead of 15%
     -- That's why it overwrites some things regular haste won't.
-    target:addStatusEffect(dsp.effect.HASTE,153,0,duration);
-    skill:setMsg(dsp.msg.basic.SKILL_GAIN_EFFECT);
-    return dsp.effect.HASTE;
+    target:addStatusEffect(dsp.effect.HASTE, 1500, 0, duration)
```

Comment:
1530?

Reply:
remove the comment about 1530, then

Reply:
Word. I'll do some research to find out if that's actually true, too.

Reply:
Hnnng.

According to BG-wiki. Haste and Hastega are both labeled 15%, but then include 150/1024 and 153/1024 respectively. This is annoying because neither of those values are 15%. They're about 14.6% and 14.9%. The whole point of this was to get accurate values while having human readable percentages in the DB for gear. I'm leaning towards adjusting the sources of magic haste to be this accurate. (So hastega would become 1494). What do you think?

Reply:
fine with me